Sources/SwiftLayout/Layout/Protocol/Deactivable.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//  
+//
+//  Created by aiden_h on 2022/02/10.
+//
+
+import Foundation
+
+public protocol Deactivable {
+    var layout: Layout { get }
```

Comment:
I don't think you need to include a layout in Deactivable protocol.

Reply:
oh I see. you right.